### ISSUE TITLE
Different grammars in the new syllabus

### ISSUE THREAD

**Issue by lightumcc** [iewnfod/CAIE_Code#58]
### 问题描述

In old syllabus (2021):
<img width="685" alt="image" src="https://github.com/iewnfod/CAIE_Code/assets/60721270/50ec5774-1ef6-41bf-8aaf-700611f25ab8">

In new syllabus (2023,2024-2025):
<img width="659" alt="image" src="https://github.com/iewnfod/CAIE_Code/assets/60721270/74d4e74b-2f94-45b4-b335-06654491459f">

### 复现方法

```
DECLARE stack : ARRAY[1:10] OF INTEGER  // Stack array with a capacity of 10 elements
DECLARE topPointer : INTEGER            // Points to the top element of the stack
DECLARE basePointer : INTEGER           // Points to the base of the stack
DECLARE stackFull : INTEGER             // Indicates the full capacity of the stack

basePointer <- 1
topPointer <- 0
stackFull <- 10

PROCEDURE popElement
    DECLARE item : INTEGER
    IF topPointer = basePointer - 1 THEN
        OUTPUT "Stack is empty"
    ELSE
        item <- stack[topPointer]
        OUTPUT item, " is popped from stack"
        topPointer <- topPointer - 1
    ENDIF
ENDPROCEDURE
```

### 版本信息

Version 0.1.4 (dev/2c0b586)
Using [ PyPy 3.10.13 ] on Darwin
Current Version Notes: Bug fix: wrong char remove

### 额外说明

[9618_pseudocode(1).pdf](https://github.com/iewnfod/CAIE_Code/files/13483953/9618_pseudocode.1.pdf)

The latest version of the syllabus has been committed on https://github.com/iewnfod/CAIE_Code/commit/be80ff72e31a922a81332c936820665251588b40

**Comment by lightumcc** [comment]
Please also check the rest of the function whether there are some changes.

**Comment by lightumcc** [comment]
No other changes have been found yet

**Comment by lightumcc** [comment]
FUNCTION
Old:
<img width="717" alt="image" src="https://github.com/iewnfod/CAIE_Code/assets/60721270/ef20037b-f11e-45dc-83ab-0ff67492daea">
New:
<img width="1101" alt="image" src="https://github.com/iewnfod/CAIE_Code/assets/60721270/94075169-dc3f-45df-80ae-0023112dc02e">

**Comment by lightumcc** [comment]
New:
<img width="757" alt="image" src="https://github.com/iewnfod/CAIE_Code/assets/60721270/c47519b5-880d-41fe-9188-b0756ac2f19b">

**Comment by iewnfod** [comment]
Difference in 8.1 & 8.2 has been fix by [dev@732379f](https://github.com/iewnfod/CAIE_Code/commit/732379f0fed9adf1642738d7bd37ab9153e8e60e)

**Comment by iewnfod** [comment]
> New: <img alt="image" width="757" src="https://private-user-images.githubusercontent.com/60721270/286869256-c47519b5-880d-41fe-9188-b0756ac2f19b.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTEiLCJleHAiOjE3MDE0OTQxNjksIm5iZiI6MTcwMTQ5Mzg2OSwicGF0aCI6Ii82MDcyMTI3MC8yODY4NjkyNTYtYzQ3NTE5YjUtODgwZC00MWZlLTkxODgtYjA3NTZhYzJmMTliLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFJV05KWUFYNENTVkVINTNBJTJGMjAyMzEyMDIlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjMxMjAyVDA1MTEwOVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTBmYzY1NTM5MjQyNGQ5OGVlZTA3ZmIyZDM1YWY1OGRiZGZhNGFlZWFkY2IwZDU4ZGEzZjc2OWQyNmU3ZTM1YmMmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.3vHHDwWJNjcDVVdN81hd01ffzfOXNC4oFu2UZw4MUEc">

I'm not sure with its meaning. Can you give me more example?
